src/dt_maps/types/watchtowers.py

```diff
@@ -0,0 +1,66 @@
+from enum import Enum
+from typing import Union, Iterable, Optional, Any
+
+from dt_maps.types.commons import EntityHelper, FieldPath
+from dt_maps.types.frames import Frame
+
+CONFIGURATION = "configuration"
+ID = "id"
+
+
+class WatchtowerType(Enum):
+    WT18: str = "WT18"
+    WT19: str = "WT19"
+
+
+class Watchtower(EntityHelper):
+
+    def _get_property_values(self, name: str) -> Optional[Iterable[Any]]:
+        return {
+            CONFIGURATION: [w.value for w in WatchtowerType],
+            ID: None
+        }[name]
+
+    def _get_property_types(self, name: str) -> Union[type, Iterable[type]]:
+        return {
+            CONFIGURATION: str,
+            ID: Optional[str]
```

Comment:
according to the return type, `Union[type, Iterable[type]]` this cannot be a Typing annotation, use `(type(None), str)` instead. These are used with `isinstance` internally, so they have to be valid types.